packages/manager/modules/netapp/src/dashboard/snapshot-policies/controller.js

```diff
@@ -0,0 +1,24 @@
+export default class OvhManagerNetAppSnapshotPoliciesCtrl {
+  $onInit() {
+    this.options = [
+      {
+        id: 1,
+        label: 'First',
+      },
+      {
+        id: 2,
+        label: 'Second',
+      },
+      {
+        id: 1,
+        label: 'Third',
+      },
+    ];
```

Comment:
Temporary item, which will be removed once the work for the whole snapshot policies page will begin